### ISSUE TITLE
Only 1000 Assest are imported to the album

### ISSUE THREAD

**Issue by ixtrader** [romainrbr/immich-face-to-album#3]
Is there a way to add more:

Added 1000 asset(s) to the album

the face has over 6000 pictures. a rerun shows:

No assets to add

**Comment by romainrbr** [comment]
Fixed with the latest version, sorry for the delay I didn't encountered this myself !

**Comment by kjdevelopers** [comment]
I too am experiencing the same issue as first reported.  Fresh install tonight and my images are capped at 1000.  Otherwise, it works great.

**Comment by romainrbr** [comment]
Hey @kjdevelopers , it should be fixed in the latest version (1.0.4). Let me know if thats not the case.

```
Total unique assets to add: 1885
Added 500 asset(s) to the album
Added 500 asset(s) to the album
Added 500 asset(s) to the album
Added 385 asset(s) to the album
```
